### ISSUE TITLE
Compilation 

### ISSUE THREAD

**Issue by imran-uk** [ssokolow/quicktile#59]
Apologies, as I know this isn't strictly a QuickTile problem...

I'm trying to get it installed on `CentOS release 6.7 (Final)`

I get the following when running quicktile.py to generate the config:

```
(quicktile.py:30274): WARNING **: Trying to register gtype 'WnckWindowState' as flags when in fact it is of type 'GEnum'

(quicktile.py:30274): WARNING **: Trying to register gtype 'WnckWindowMoveResizeMask' as flags when in fact it is of type 'GEnum'

File "./quicktile.py", line 657, in WindowManager
    wnck.WINDOW_CHANGE_WIDTH | wnck.WINDOW_CHANGE_HEIGHT):
    AttributeError: 'module' object has no attribute 'WINDOW_CHANGE_X'
```

I have all the needed libs AFAICT but maybe I am missing some minimum version?

```
Package python-2.6.6-64.el6.x86_64 already installed and latest version
Package pygtk2-2.16.0-3.el6.x86_64 already installed and latest version
Package pygobject2-2.20.0-5.el6.x86_64 already installed and latest version
Package dbus-python-0.83.0-6.1.el6.x86_64 already installed and latest version
Package gnome-python2-libwnck-2.28.0-5.el6.x86_64 already installed and latest version
```

In Googling this issue I tried applying a [patch](https://bugzilla.redhat.com/show_bug.cgi?id=643930) to gnome-python2-libwnck-2.28 but that just produced a different error.

QuickTile is a "must have" for me when working in Gnome 2/MATE and Centos 6 is still fully supported by Red Hat - so there is likely others with the same issue and I'd appreciate this left open in case someone see's this and has fixed it before. Thank

**Comment by ssokolow** [comment]
According to your link, it's a bug that prevents bits of libwnck 2.28 from being exposed to Python programs.

Unfortunately, without those two flags, QuickTile cannot request that windows be resized (which makes it rather useless) and I switched to libwnck because I got sick of tearing my hair out trying to reinvent it.

**Comment by psrivongse** [comment]
@imran-uk Have you figured out a solution to your problem or have any other similar applications that you might've found that works with Centos 6.7? I'm in the same boat and haven't been able to get a proper window tiling application to work.

**Comment by imran-uk** [comment]
@psrivongse sadly I did not find a solution and I think we're moving to CentOS 7 soon so I lost the will to pursue it further.

What I did instead was to install and try out [Terminator](http://gnometerminator.blogspot.co.uk/p/introduction.html) which I now like very much and gives me 80% of what I was looking for. I strongly encourage you to install and try it. It is available for CentOS 6 and works well.
